### DISCUSSION TITLE
FE single view: Which features do you use?

### DISCUSSION THREAD

**Discussion by oliverklee** [oliverklee/ext-seminars#1381] [Ideas]
As part of a customer project, I will probably complete rewrite the FE single/detail view. For this, I plan to only implement the functionality, displayed and hooks that are actually still used/needed. For this, I need your input. :-)

Please comment:
- Which display field in the FE single view do you use?
- Which of the FE list single hooks do you use? For which purpose?
- Do you use the "other dates for this event" feature?

**Comment by lisardo2** [comment]
Which display field in the FE single view do you use?
- title
- description
- target group
- event type
- date
- price regular
- price special
- place
- leaders
- organizers
- payment method
- deadline registration
- registration

Which of the FE list single hooks do you use? For which purpose?
- hook for custom marker

Do you use the "other dates for this event" feature?
- no

**Reply by jpmschuler** [reply]
As we just have the most basic use of fe_login, select event (no use of actual dates, prices or anything), add notes (no fields except a free field) and join waiting queue, thus:
We only use `title`,`date`,`place` - however often the latter to has just the start and end like `1st Aug-16 Dec` and place is often `tba, refer to event details` as the other stuff is defined in other custom records (especially start date end date, notification dates, weekly dates and rooms) as we have a lot of events which don't need registration and want thus EXT:seminars is used for registration only.

**Comment by peerlight** [comment]
## Which display field in the FE single view do you use?

- in  fact "all", as we only excluded credit_points, language, eventsnextday

## Which of the FE list single hooks do you use? For which purpose?

- none

## Do you use the "other dates for this event" feature?

- no

**Comment by oliverklee** [comment]
@jpmschuler Thanks! Just to be sure: You only display title, date and place in the FE single view?

**Reply by jpmschuler** [reply]
Exactly (and the register link at the bottom of not yet registered)

**Comment by oliverklee** [comment]
From a customer:
- title
- date + time
- place
- vacancies
- price
- register link

**Comment by oliverklee** [comment]
From another customer:
- title
- subtitle
- date + time
- time slots
- price
- place
- vacancies
- max. seats
- registration link
- speakers
- target groups
